docs/node.md

```diff
@@ -0,0 +1,103 @@
+# Node
+
+The Node language dependency checks logic consists of two separate files, similar to other languages: a bash script and a Dockerfile.
+
+The **bash script** performs dependency checking by fetching project details (such as folder path and file name) from the dependency state file. It then reports the check results to the reports folder. The dependency check tool we are using in this context is npm audit.
+
+One way node potentially differs from other languages is: the possibility of there being a package.json and a package-lock.json. Leading to two possibilities where you either do npm install or npm ci.
```

Comment:
Add backticks around npm commands

Reply:
Thanks @Harrisman05, I will do it in the next PR, I will approve it, could you please merge it